Makefile

```diff
@@ -370,3 +371,13 @@ certs-dryrun:
 .PHONY: checkdoc
 checkdoc:
 	go run cmd/checkdoc/main.go cmd/checkdoc/docs.go --project-path $(CURDIR) --component-rel-path $(COMP_REL_PATH) --module-name $(MOD_NAME)
+
+# Construct new API state snapshots
+.PHONY: apidiff-build
+apidiff-build:
+	@$(foreach pkg,$(ALL_PKGS),$(call exec-command,./internal/buildscripts/gen-apidiff.sh -p $(pkg)))
+
+# Compare API state snapshots
+.PHONY: apidiff-compare
+apidiff-compare:
+	@$(foreach pkg,$(ALL_PKGS),$(call exec-command,./internal/buildscripts/compare-apidiff.sh -p $(pkg)))
```

Comment:
Nit: add a new line at the end to please github diff viewer.

Reply:
Fixed! Usually I'm the one calling out missing newlines at EOF, so this is a little :embarrassing: